rakelib/release.rake

```diff
@@ -134,27 +153,48 @@ namespace :release do
     JSON.parse(response.body)
   end
 
+  def github_upload(json, filename)
+    upload_url = json['upload_url'].gsub(/\{.*\}/, "?name=#{filename}")
```

Comment:
I think it would make it a bit less obscure to pass the upload_url directly to `github_upload`?
Also, I didn't know better back when I wrote that Rakefile a while ago, but nowadays I prefer using named parameters for readability… which would thus look like:

```suggestion
  def github_upload(upload_url:, filename:)
    upload_url = upload_url.gsub(/\{.*\}/, "?name=#{filename}")
```
And we'd then adjust the call site [below](https://github.com/SwiftGen/SwiftGen/pull/926/files#diff-1e28be9fbab9e305fe4d9e1bbdfeca0bbe325cb72a926b17f464f745dc8d8cc4R197) accordingly. wdyt?